scripts/test-run-sqlite

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 start-test() {
-    KINE_IMAGE=$IMAGE KINE_ENDPOINT="sqlite" provision-kine
+    KINE_IMAGE=$IMAGE KINE_ENDPOINT="" provision-kine
```

Comment:
`[sqlite] time="2024-09-03T21:27:16Z" level=fatal msg="flag needs an argument: -endpoint"`

You can leave `--endpoint` unset, but you can't currently set it to an empty string... so you should either fix the CLI flag, or explicitly set it.

```suggestion
    KINE_IMAGE=$IMAGE KINE_ENDPOINT="sqlite://" provision-kine
```

Reply:
Actually I suspect that when the var is empty the script is just running `kine --endpoint` instead of `kine --endpoint ""` so the CLI framework is complaining that it doesn't have a value - which is valid.